tests/std/tests/VSO_0000000_instantiate_iterators_misc/test.compile.pass.cpp

```diff
@@ -17,6 +17,7 @@
 #define _SILENCE_CXX20_OLD_SHARED_PTR_ATOMIC_SUPPORT_DEPRECATION_WARNING
 #define _SILENCE_CXX20_REL_OPS_DEPRECATION_WARNING
 #define _SILENCE_CXX20_U8PATH_DEPRECATION_WARNING
+#define _SILENCE_CXX20_CISO646_REMOVED_WARNING

```

Comment:
Why does this test include `<ciso646>` in the first place? Does it define templates that we need to instantiate, or could we simply strike the `#include`?

Reply:
the test passes without the `#include`

Reply:
I suspect the include was just an artifact of the original test writer including _everything_.